packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 ### New features and enhancements
 
+- **Breaking:** Made `MainframeInteraction.IJes.getJobsByParameters` a required function now that `getJobsByOwnerAndPrefix` has been removed.
```

Comment:
please add a PR/issue link. can we explain what requires this function, and why other getJobsetc was removed?

Reply:
Another contributor had removed it in a previous snapshot in favor of the new API. I've updated the changelog to mention the reason for the removal. I've also added a PR link in my latest commit and adjusted the wording to clarify what requires this function (the `MainframeInteraction.IJes` interface).